### PR TITLE
SLUDGE: Fix drawModeDark colourmix values by swapping them

### PR DESCRIPTION
**Pull request by polyesterswing**
In Out of Order, if you visit the alley that goes to the surgery you will notice a jarring transition between colours on Hurford.

### CODE REVIEW

**Comment by bluegr** [comment]
I don't understand. These values are swt exactly like the original sources:
https://github.com/opensludge/opensludge/blob/a247b3669d41b3c2de0d06b88fc3e21ad4898a47/source/Engine/people.cpp#L281

Why change them? The issue must be somewhere else

**Comment by polyesterswing** [comment]
@bluegr  This issue can be seen with games in the original engine as well.
Look at  this commit [https://github.com/opensludge/opensludge/commit/3ecf69a0f0fb17f176d0882fd7a347a510a0903f](url)
The values for the `drawModeDark` have not been changed correctly.
It seems that games released on the original engine that don't have this commit behave differently.
The documented effects can be seen under [https://opensludge.github.io/opensludge/doc/SLUDGEDevKitHelp/setCharacterDrawMode.html]()

**Comment by bluegr** [comment]
In this case, some older games should use the older semantics, and newer ones should use the newer values. We should change these values only for the older games that are affected, by using a feature flag in their detection entries, so that the newer games won't be affected by this change

File: engines/sludge/people.cpp [hunk 1]
```diff
@@ -333,6 +333,9 @@ enum drawModes {
 };
 
 void PeopleManager::setMyDrawMode(OnScreenPerson *moveMe, int h) {
+	bool hasBadDrawMode = gameVersion > VERSION(1, 7);
```
Comment (sev-) [inline engines/sludge/people.cpp:336]: This is not portable. `int drawMode = (gameVersion > VERSION(1, 7)) ? 1 : 0`

File: engines/sludge/people.cpp [hunk 2]
```diff
@@ -333,6 +333,9 @@ enum drawModes {
...
+	const int colourMix[2][3] = { {64, 128, 192}, {192, 128, 64}};
```
Comment (bluegr) [inline engines/sludge/people.cpp:337]:
```suggestion
	const int colourMix[2][3] = { {64, 128, 192}, {192, 128, 64} };
```

**Comment by sev-** [comment]
Thank you!

**Pull request merged**